src/core/components/tabs/tabsTrigger/tabsTrigger.test.tsx

```diff
@@ -0,0 +1,42 @@
+import { render, screen } from '@testing-library/react';
+import { IconType } from '../../icon';
+import { Tabs, type ITabsTriggerProps } from '../../tabs';
+
+describe('<Tabs.Trigger /> component', () => {
+    const createTestComponent = (props?: Partial<ITabsTriggerProps>, isUnderlined?: boolean) => {
+        const completeProps: ITabsTriggerProps = {
+            label: 'Tab 1',
+            value: '1',
+            ...props,
+        };
+
+        return (
+            <Tabs.Root isUnderlined={isUnderlined}>
+                <Tabs.List>
+                    <Tabs.Trigger {...completeProps} />
+                </Tabs.List>
+            </Tabs.Root>
+        );
+    };
+
+    it('should render without crashing', () => {
+        render(createTestComponent());
+
+        expect(screen.getByText('Tab 1')).toBeInTheDocument();
```

Comment:
Check for the presence of the trigger instead by using `getByRole`. I believe the trigger is a button. Relying on the default text isn't ideal.